packages/permission_handler/tizen/src/permission_manager.cc

```diff
@@ -219,75 +155,145 @@ int PermissionManager::DeterminePermissionStatus(int permission, int* status) {
       switch (check_result) {
         case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_DENY:
         case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK:
-          *status = PERMISSION_STATUS_DENIED;
+          *status = PermissionStatus::kDenied;
           return result;
         case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ALLOW:
         default:
-          *status = PERMISSION_STATUS_GRANTED;
+          *status = PermissionStatus::kGranted;
           break;
       }
     }
   }
   return result;
 }
 
-void PermissionManager::OnRequestPermissionsResponse(
-    ppm_call_cause_e cause, const ppm_request_result_e* results,
-    const char** privileges, size_t privileges_count, void* user_data) {
-  if (!user_data) {
-    LOG_ERROR("Invalid user data");
+void OnRequestPermissionsRespon(ppm_call_cause_e cause,
```

Comment:
Is there any special reason why you shortened the name of this function?

Reply:
Could you please refer to the conversation containing this [comment](https://github.com/flutter-tizen/plugins/pull/139#discussion_r660213565)? 
When I changed this lambda function back to a named function, I made it a local function and excluded it from the class namespace.

Reply:
OnRequestPermissions**Response** ->OnRequestPermissions**Respon**
Oh!? Are you talking about this? My mistake! I'll fix it later when I get to work.